plugins/techdocs-node/api-report.md

```diff
@@ -248,5 +251,7 @@ export const transformDirLocation: (
 export class UrlPreparer implements PreparerBase {
   static fromConfig({ reader, logger }: PreparerConfig): UrlPreparer;
   prepare(entity: Entity, options?: PreparerOptions): Promise<PreparerResponse>;
+  // (undocumented)
```

Comment:
Could you [inherit](https://api-extractor.com/pages/tsdoc/tag_inheritdoc/) the documentation here?